modules/ztp-install-openshift-pipelines.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: PROCEDURE
+[id="install-the-openshift-pipelines-operator_{context}"]
+= Installing the {product-title} Pipelines Operator
+
+Follow this guidance to install the OpenShift Pipelines Operator that is used to run the pipeline.
```

Comment:
The heading calls it the "{product-title} Pipelines Operator", but here just says "OpenShift Pipelines Operator". Is one more correct, and should it be consistent throughout?